websocket_test.go

```diff
@@ -765,17 +779,36 @@ const serviceWebSocket = "WebSocket"
 
 type ServiceWebSocket struct {
 	*ServiceProcessor
+	Errors int
```

Comment:
do you need a lock for accessing this field?

Reply:
No, because golang doesn't complain.

I meant: no, because it's only accessed once per call, and each service has its own `Errors`. Also, the `SendProtobufParallel` call should be done when the counting starts. So a golang race-error would be an indication of the call not having finished.